tests/foreman/api/test_capsulecontent.py

```diff
@@ -1894,3 +1894,63 @@ def test_positive_reclaim_space(
         assert (
             reclaim_doc['apis'][0]['api_url'] == '/katello/api/capsules/:id/content/reclaim_space'
         ), 'Documented path did not meet the expectation.'
+
+
+class TestPodman:
+    """Tests specific to using podman push/pull on Satellite
+
+    :CaseComponent: Repositories
+
+    :team: Phoenix-content
+    """
+
+    @pytest.fixture(scope='class')
+    def enable_podman_capsule(module_product, module_capsule_configured):
+        """Enable base_os and appstream repos on the sat through cdn registration and install podman."""
+        module_capsule_configured.register_to_cdn()
+        if module_capsule_configured.os_version.major > 7:
+            module_capsule_configured.enable_repo(module_capsule_configured.REPOS['rhel_bos']['id'])
+            module_capsule_configured.enable_repo(module_capsule_configured.REPOS['rhel_aps']['id'])
+        else:
+            module_capsule_configured.enable_repo(module_capsule_configured.REPOS['rhscl']['id'])
+            module_capsule_configured.enable_repo(module_capsule_configured.REPOS['rhel']['id'])
+        result = module_capsule_configured.execute(
+            'dnf install -y --disableplugin=foreman-protector podman'
+        )
+        assert result.status == 0
```

Comment:
I've seen this fixture at two other places. Would it be possible to make it rather a method in the `Satellite` or `Capsule` class? Or move it to `robottelo/pytest_fixtures/core` instead?

Reply:
Going to tackle this in a separate PR, for all of these fixtures I've introduced at the same time.

Reply:
ACK, tracked by https://github.com/SatelliteQE/robottelo/issues/15953